index.php

```diff
@@ -1,17 +1,20 @@
 <?php
 require_once('functions.php');
+$add_entry_message = '';
+if (!empty($_GET)) {
```

Comment:
same coment as before, you need to specifically check if the key is empty, not just GET